### PR TITLE
Check scoped_services before resolving from map when in a scope

### PR DESCRIPTION
**Pull request by StummeJ**
I thought it was really odd to have scoped_services, but not have a way to get it through the provider's get method. This leads to needing to know you're in a scope when trying to resolve a dependency type, which is not the best practice if you wish to encapsulate things.

### CODE REVIEW

**Comment by RobertoPrevato** [comment]
Hi @StummeJ 
Thank You for your contributions, and sorry for taking so long to reply. I am willing to merge.

File: tests/test_services.py [hunk 1]
```diff
@@ -483,6 +483,24 @@ def test_scoped_services():
     assert b is not d
 
 
+def test_scopeed_service_from_scoped_services():
```
Comment (RobertoPrevato) [inline tests/test_services.py:486]: Please fix the typo (`scopeed` instead of `scoped`).

**Comment by StummeJ** [comment]
No worries, I know people can get busy with life. Thanks!

File: tests/test_services.py (hunk 1 above)
Comment (StummeJ) [reply tests/test_services.py]: Fixed and ran `make format`

**Comment by RobertoPrevato** [comment]
@StummeJ Thank You!

**Pull request merged**